include/suit_types.h

```diff
@@ -74,7 +74,9 @@ enum suit_command_sequence {
 /** https://www.iana.org/assignments/cose/cose.xhtml */
 enum suit_cose_alg {
 	suit_cose_sha256 = -16,
-	suit_cose_es256 = -7,
+	suit_cose_es256 = -7, /* Deprecated */
```

Comment:
Why `Deprecated`?

Reply:
I found that information while looking for values for cose algorithms. I can't find that document again and the one that's recently updated (https://www.iana.org/assignments/cose/cose.xhtml) has no such information so that's probably my mistake. What's more I can't find value -50 so that probably is also wrong. 
Removing comments and enum suit_cose_Ed25519.